docs/source/debugging.rst

```diff
@@ -2,18 +2,18 @@ Debugging
 =========
 The following are flags that make debugging much easier.
 
-Fast dev run
-------------
-This flag runs a "unit test" by running 1 training batch and 1 validation batch.
-The point is to detect any bugs in the training/validation loop without having to wait for
+Unit test
+---------
+This flag runs a "unit test" by running 1 training batch, 1 validation batch and 1 test batch.
+The point is to detect any bugs in the training/validation/test loop without having to wait for
 a full epoch to crash.
 
-(See: :paramref:`~pytorch_lightning.trainer.trainer.Trainer.fast_dev_run`
+(See: :paramref:`~pytorch_lightning.trainer.trainer.Trainer.unit_test`
```

Comment:
this shall be `.. seealso::` @awaelchli right?

Reply:
I am not quite sure i understand the recommended change. I believe i need to add reference of Trainer module? Is my understanding correct ?

Reply:
use `.. seealso::` intead `(See: `